### DISCUSSION TITLE
Use CachedEventHandler options in nitro routes config

### DISCUSSION THREAD

**Discussion by LouisHaftmann** [nitrojs/nitro#442] [Q&A] [answered]
It is already possible to enable `SWR` by passing `swr: true` option in nitro routes config but you can't set `maxAge` or any other `CachedEventHandler` options.

Example usage:
```typescript
// nitro.config.ts
import { defineNitroConfig } from 'nitropack'
export default defineNitroConfig({
  routes: {
    '/blog/**': { swr: true, maxAge: 60 }
  }
})
```

**Comment by Hebilicious** [comment, marked as answer]
> It is already possible to enable `SWR` by passing `swr: true` option in nitro routes config but you can't set `maxAge` or any other `CachedEventHandler` options.
> 
> Example usage:
> 
> ```ts
> // nitro.config.ts
> import { defineNitroConfig } from 'nitropack'
> export default defineNitroConfig({
>   routes: {
>     '/blog/**': { swr: true, maxAge: 60 }
>   }
> })
> ```

You can specify cache options in RouteRules https://nitro.unjs.io/guide/cache#route-rules
